src/Tizen.NUI/src/internal/AccessibilityDoGestureSignal.cs

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright(c) 2019-2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.Diagnostics;
+using System;
+using System.Drawing;
+
+namespace Tizen.NUI
+{
+    using global::System;
+    using global::System.ComponentModel;
+    using global::System.Runtime.InteropServices;
+    using Tizen.NUI.BaseComponents;
+
+    public class AccessibilityDoGestureSignal : Disposable
+    {
+        internal AccessibilityDoGestureSignal(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn) {
+        }
+
+        static internal int ConvertParam1(global::System.IntPtr data) {
```

Comment:
It is internal API, but I think it would be better to know what it converts.

Reply:
Done.